src/lib/operators/jit_operator/operators/jit_read_tuples.cpp

```diff
@@ -164,16 +328,6 @@ JitTupleEntry JitReadTuples::add_literal_value(const AllTypeVariant& value) {
 }
 
 JitTupleEntry JitReadTuples::add_parameter(const DataType data_type, const ParameterID parameter_id) {
```

Comment:
Code removed as every access to a parameter must have its own tuple entry now. If the same parameter is used in different expressions using value ids, different entries are requires to hold the different value ids.